index.js

```diff
@@ -2,15 +2,15 @@ const streamToPromise = require('stream-to-promise')
 const rp = require('request-promise')
 
 const url = 'https://graph-video.facebook.com'
-const version = 'v3.2'
+const version = 'v9.0'
 
 const retryMax = 10
 let retry = 0
 
 function apiInit(args, videoSize) {
   const options = {
     method: 'POST',
-    uri: `${url}/${version}/${args.id}/videos?access_token=${args.token}`,
+    uri: `${url}/${version}/${args.id}/advideos?access_token=${args.token}`,
```

Comment:
`videos` and `advideos` are not the same endpoint.
I didn't know the api was similar for this endpoint maybe it can be an option with the default set to `videos`.